Added section with operators and custom operators.

AndersHqst opened the pull request.

AndersHqst committed:
Added section with operators and custom operators.

AndersHqst commented:
@ssnielsen will you give this a look? :)

ssnielsen commented on 2015-Fall/andershqst/SwiftLanguage/SwiftLanguage.playground/Contents.swift:
```diff
@@ -478,3 +478,115 @@ struct StaticExample2 {
+// myBool &&= true // Logical AND (Not working in playground?)
+// myBool ||= false // Logical OR (Not working in playground?)
+
+// Postfix
```
Already mentioned further up :)

AndersHqst replied:
Yearh I know. Kind of got into the mode that it would be nice to have them listed in one place. What do you think?

ssnielsen replied:
Definitely in the same place.

ssnielsen commented:
Seems like a good overview of the operators. Although to me the order in which they are presented (wrt. their 'complexity') seems a bit off to me. I would expect arithmetic operators together with assignment to be the first. And maybe more clear 'categories' of the operations ie. arithmetic, boolean/logic, bitwise etc. Maybe using the same order as in the swift book?

AndersHqst committed:
Moved simple stuff to the top. Removed comments and too "theory-like" naming. Bitwise, overflow-ignoring, and custom operators are left at the tail of the playground

AndersHqst commented:
Yes, had the same gut feeling. I have moved it around now and deleted some dublicate stuff. All simple operators that one would now already is now put at the top and grouped in a more simple way, with less comments and theoretical naming.

AndersHqst committed:
Added more comments
